rocAL/rocAL/source/meta_data/meta_node_crop_resize.cpp

```diff
@@ -73,11 +80,14 @@ void CropResizeMetaNode::update_parameters(pMetaDataBatch input_meta_data, pMeta
                 float yA = std::max(crop_box.t, box.t);
                 float xB = std::min(crop_box.r, box.r);
                 float yB = std::min(crop_box.b, box.b);
-                box.l = (xA - crop_box.l) / (crop_box.r - crop_box.l);
-                box.t = (yA - crop_box.t) / (crop_box.b - crop_box.t);
-                box.r = (xB - crop_box.l) / (crop_box.r - crop_box.l);
-                box.b = (yB - crop_box.t) / (crop_box.b - crop_box.t);
-                
+                box.l = (xA - crop_box.l);
+                box.t = (yA - crop_box.t);
+                box.r = (xB - crop_box.l);
+                box.b = (yB - crop_box.t);
+                box.l *= _dst_to_src_width_ratio;
+                box.t *= _dst_to_src_height_ratio;
+                box.r *= _dst_to_src_width_ratio;
+                box.b *= _dst_to_src_height_ratio;
```

Comment:
```suggestion
                box.l = (xA - crop_box.l);
                box.t = (yA - crop_box.t);
                box.r = (xB - crop_box.l);
                box.b = (yB - crop_box.t);
                box.l *= _dst_to_src_width_ratio;
                box.t *= _dst_to_src_height_ratio;
                box.r *= _dst_to_src_width_ratio;
                box.b *= _dst_to_src_height_ratio;
```

We can do all this in just 4 lines right? Why do subtraction and then multiply separately?